tasmota/support_switch.ino

```diff
@@ -327,31 +329,49 @@ void SwitchHandler(uint8_t mode)
           break;
         case PUSHHOLDMULTI:
           if (NOT_PRESSED == button) {
-            if (Switch.hold_timer[i] != 0) {
+            if ((Switch.hold_timer[i] & 0x3F) != 0) {
```

Comment:
Wouldn't it be better to use a constant with a meaningful name for the numeric value so that the code is more readable?

Reply:
like 0b00111111  ???
I am used to hex numbers for binary operations, but binary code is olso ok for me

Reply:
No, that is just as little self-explanatory. What does ```0x3F``` stand for, so what does the value say?

Reply:
ok.     
For 0x3F I could use TIMER_MASK   
for 0xFF NO_TIMER_MASK
for 0x40    FIRST_PRESS
for 0x80    SECOND_PRESS
thats maybe better to read

Reply:
Pls do and prefix them with ``SM_`` to make them standout of other global defines.

Waiting for your updated PR.